controller/src/main/java/io/pravega/controller/store/PravegaTablesStoreHelper.java

```diff
@@ -244,6 +249,89 @@ public void invalidateCache(String table, String key) {
                 .whenComplete((r, ex) -> releaseEntries(entries));
     }
 
+    public CompletableFuture<List<Version>> removeValues(final String tableName, Set<String> tableKeys, final String appendValue, long requestId) {
+        Supplier<String> errorMessage = () -> String.format("remove values: on table: %s", tableName);
+
+        // read values and add values
```

Comment:
nit: this comment says read and add while the call below is read and remove.

Reply:
fixed it.